srcpkgs/quixand/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'quixand'
+pkgname=quixand
+version=2012.12.24.1
+revision=1
+noarch=yes
+depends="bash encfs"
+short_desc="A tool for creating single-use unrecoverable encrypted sandboxes"
+maintainer="Andy Weidenbaum <archbaum@gmail.com>"
+license="GPL-2.0-only"
+homepage="https://xyne.archlinux.ca/projects/quixand/"
+distfiles="https://xyne.archlinux.ca/projects/quixand/src/${pkgname}-${version}.tar.xz"
+checksum=3d8f1b03de824d5b9357b5479ecdfcdce22985671eac5234199fb4504cb2ba11
+
+do_install() {
+	vbin quixand
+}
+
+post_install() {
```

Comment:
no need for separate post_install() if you're defining your own do_install, move it over there